lib/transformers/model.schema.transformer.js

```diff
@@ -47,9 +49,33 @@ function _fillRelationshipFields(model, processedSchema, opts) {
 }
 
 function _resolveSchemaForRelation(processedClass, relationEntry, opts) {
-  const relatedClass = _.isString(relationEntry.modelClass)
-    ? require(relationEntry.modelClass)
-    : relationEntry.modelClass;
+  const modelPaths = processedClass.modelPaths;
+  let relatedClass;
+
+  if (_.isString(relationEntry.modelClass)) {
```

Comment:
there seems to be some duplicate code here. can some or all of it be moved into a shared util file?

Reply:
sure

Reply:
Done